src/services/Authentications.ts

```diff
@@ -24,31 +55,28 @@ export default class Authentication {
   static async login({ email, password }) {
```

Comment:
You should expect the express res object to come in as a parameter in your function.